packages/server/api/controllers/movies.controller.js

```diff
@@ -50,10 +50,39 @@ const createMovie = async (body) => {
   };
 };
 
+// get  favorite movies by userID
+const getFavoriteMovie = async (userID) => {
+  return knex('movies')
+    .select('*')
+    .join('favorites', 'favorites.user_id', '=', 'movies.id')
+    .where({ user_id: userID });
+};
+
+// add favorite movie
+const addFavoriteMovie = async (body) => {
+  await knex('favorites').insert({
+    movie_id: body.movie_id,
+    user_id: body.user_id,
+    comment: body.comment,
+  });
+
+  return {
+    successful: true,
```

Comment:
May be return id of the favourite movies that you just added. And appropriate http status like 201(CREATED)?